Buttplug.Core/IButtplugLog.cs

```diff
@@ -25,11 +25,5 @@ public interface IButtplugLog
 
         [CanBeNull]
         event EventHandler<LogExceptionEventArgs> OnLogException;
-
-        [NotNull]
-        Error LogErrorMsg(uint aId, ErrorClass aCode, string aMsg);
-
-        [NotNull]
-        Error LogWarnMsg(uint aId, ErrorClass aCode, string aMsg);

```

Comment:
Made these methods as Extension methods to reduce copying code and only needing to implement whjat's required